web/src/pages/news/blogs/index.page.test.tsx

```diff
@@ -0,0 +1,313 @@
+import { ParsedUrlQuery } from "querystring";
+
+import { getStoryblokApi } from "@storyblok/react";
+import { render, screen, waitFor, within } from "@testing-library/react";
+import { useRouter } from "next/router";
+import { GetServerSidePropsContext } from "next/types";
+import { StoryblokStory } from "storyblok-generate-ts";
+
+//TODO is separate mock data necessary? Authors aren't coming through - how to handle this?
```

Comment:
Anything more you wanted to do here or are you happy to leave the TODO in place? Can't quite suss out what the issue is from the comment

Reply:
I think the snapshots cover the behaviour as we can't very well mock the way storyblok behaves (where it gets authors with the resolve relationships behaviour. Happy to remove this.